### DISCUSSION TITLE
Battery/props/motors alternatives.

### DISCUSSION THREAD

**Discussion by santiagorg2401** [bitcraze/discussions#63] [Q&A]
Hi,
We are currently making a shopping list to renovate our Crazyflie fleet supplies as batteries, propellers and motors, because they are reaching the end of their life span, we are looking to improve our batteries capacity without affecting the overall on board balance, also, we are looking for more durable propeller options as they break too often, finally, we think the motors are getting worn out too, the flight is significantly erratic (we use LPS), so, any insight would be really helpful.
Thank you in advance.

**Comment by knmcguire** [comment]
Just for reference! This originates from the following forum discussion: https://forum.bitcraze.io/viewtopic.php?t=5244

**Comment by tobbeanton** [comment]
I wish I had a good answer to those questions but so far we have not found any obviously improved motors, batteries or props. If you do find that please let us know. If you want a battery with better "umpf" you can use the Tattu 350mAh 1S battery. It is a bit thicker so longest pin headers might have to be used. Also it does not have a protection circuit (PCM) so it has to be handled more carefully and don't let it deplete as it doesn't have a under-voltage protection.

**Reply by santiagorg2401** [reply]
Alright, thank you.
